tests/sentry/tasks/test_taskworker_rollout.py

```diff
@@ -39,7 +39,7 @@ def test_with_taskworker_rollout(self, mock_send_task, mock_random):
             name="test.test_with_taskworker_rollout",
             taskworker_config=self.config,
         )
-        def test_task() -> None:
```

Comment:
might be worth to rename as `placeholder_task` so it's clearer for automation tooling that these aren't tests